web/src/theme/_jobs.scss

```diff
@@ -1,3 +1,12 @@
 .multiline {
-  white-space: pre;
+  overflow-wrap: break-word;
+  white-space: pre-line;
+  width: 500px;
```

Comment:
nope :(
Hard-coded fixed widths in pixels is not a good idea. Simply resize the browser window to make it smaller to see why this is bad.